e2e/nextjs/package.json

```diff
@@ -5,11 +5,13 @@
 	"scripts": {
 		"dev": "next dev -p 3005",
 		"build": "next build",
-		"deploy": "yarn build && npx vercel build && npx vercel deploy --prebuilt",
+		"deploy": "yarn build && vercel build && vercel deploy --prebuilt",
 		"start": "next start",
-		"vercel:deploy": "yarn build && vercel build && vercel deploy --prebuilt",
 		"lint": "next lint"
 	},
+	"installConfig": {
+		"hoistingLimits": "workspaces"
+	},
```

Comment:
this seems to be breaking the automatic vercel PR preview deploy

Reply:
That's strange, because it fixes my local deploy.

One more mystery to untangle! 

Thanks for catching this.

Reply:
@Vadman97 I did some quick looking, and I'm missing the connection. Did you see an error somewhere that queued you into `hoistingLimits` as the reason behind the failure? It looks like it was having trouble finding the an unrelated file.

![image](https://github.com/highlight/highlight/assets/878947/afa75f3d-9290-4038-830a-b96404b35d40)

`hoistingLimits` are localized to the folder setting them. `e2e/nextjs` shouldn't affect `highlight.io`